3.processing_features/2.pycytominer_singlecell_pipelines.ipynb

```diff
@@ -157,8 +157,8 @@
      "text": [
       "Now performing single-cell pycytominer pipeline for Plate_6\n",
       "Annotated dataframe shape (7383, 2319)\n",
-      "Performing normalization for Plate_6 using samples parameter: all\n",
-      "Aggregated dataframe shape (60, 1128)\n"
+      "Performing normalization for Plate_6 using samples parameter: Metadata_Institution == 'iNFixion' and (Metadata_genotype == 'Null' or Metadata_genotype == 'WT')\n",
+      "Aggregated dataframe shape (20, 1138)\n"
```

Comment:
Hmmm, i don't think this should have updated the shape... all this should do is to identify the transformation based on the samples parameter. The data shape should be the same... please double check!

Reply:
I checked and it looks like there could be a bug in the "samples" parameter. I am going to check internally with the team to see if there is a fix or if this is a bug. I will be blocked on this until then.

Reply:
Bug has been fixed and looks like output is corrected!